client/components/dialog/README.md

```diff
@@ -142,11 +146,13 @@ can `@extend` the base `dialog` SCSS classes if you just want to tweak things a
 (not the backdrop)
 
 ```js
-	render: function() {
+	render {
+		const { translate } = this.props;
+
 		return (
-			<Dialog baseClassName="custom-dialog" additionalClassNames="critical error" isVisible={ this.state.showDialog } onClose={ this._onCloseDialog }>
-				<h1>{ this.translate( 'Dialog Title' ) }</h1>
-				<p>{ this.translate( 'Dialog content' ) }</p>
+			<Dialog baseClassName="custom-dialog" additionalClassNames="critical error" isVisible={ this.state.showDialog } onClose={ this.onCloseDialog }>
```

Comment:
Another thing that's not strictly in scope is that this line looks long and could benefit from being broken out on to multiple lines (FFTI)

Reply:
would be cool if we could apply prettier to jsx in markdown 🤔